### DISCUSSION TITLE
Theme setup

### DISCUSSION THREAD

**Discussion by elasti-co** [filamentphp/filament#7815] [Help]
Hi everyone,

Does anyone knows how to properly customise theme in Filament 3?

I run the command `php artisan make:filament-theme` and got almost empty theme files.
Are there any variables? I would like to make all text larger and less bold. Do I need to inspect all the classes of tables, buttons, navigations and apply (override) tailwind classes?

**Comment by leandrocfe** [comment]
yes, you need to inspect the classes and override what you want.
I did some customizations here (in pt-br) 

https://youtu.be/JvBOr_VLCm0?t=194
https://gist.github.com/leandrocfe/5336aa749a606d3c6c2c39bd3198db78

**Comment by danharrin** [comment]
https://filamentphp.com/docs/3.x/support/style-customization

**Comment by elasti-co** [comment]
I see, thank you.
I just thought that it would be useful to setup basic things like base font-size, paddings and margins in tailwind.config.js theme.

**Reply by danharrin** [reply]
You can do that if you want, if you read the Tailwind docs about it?

**Comment by elasti-co** [comment]
Thank you, will dig deeper to learn how to do it 👨🏼‍💻
